controller/src/conf/controller.config.properties

```diff
@@ -68,6 +68,13 @@ controller.retention.frequency.minutes=${RETENTION_FREQUENCY_MINUTES}
 controller.retention.bucket.count=${BUCKET_COUNT}
 controller.retention.thread.count=${RETENTION_THREAD_POOL_SIZE}
 
+controller.retention.type=${RETENTION_TYPE}
+# If retention type is Time then min value and max value are in minutes
+# If retention type is Size then min value and max value are in bytes
+controller.retention.min.value=${RETENTION_MIN_VALUE}
+controller.retention.max.value=${RETENTION_MAX_VALUE}
+controller.retention.global.policy=${GLOBAL_RETENTION_POLICY}
```

Comment:
This can be changed to "**controller.retention.global.policy.enable=${GLOBAL_RETENTION_POLICY_ENABLED}**" as this param is of type Boolean.

Reply:
Done